__tests__/testFunc/auth.js

```diff
@@ -0,0 +1,195 @@
+const request = require('supertest');
+const user = require('../util/test-user');
```

Comment:
you do require('../util/test-user') in every test. When you require this file, you create a new user and save it. It means, that every time you do require('../util/test-user'); you create new user. You should create test user only ones, and use him everywhere